src/App.jsx

```diff
@@ -13,15 +13,115 @@ function App() {
     const [todoList, setTodoList] = useState([]);
     const [isLoading, setIsLoading] = useState(true);
 
-    useEffect(() => {
-        new Promise((resolve, reject) => {
-            setTimeout(() => {
-                resolve({ data: { todoList: [] } });
-            }, 2000);
-        }).then((result) => {
-            setTodoList(result.data.todoList);
+    // Get tasks from AirTable
+    const fetchData = async () => {
+        const apiURL = `https://api.airtable.com/v0/${
```

Comment:
You noted in your submission that you wanted to pass in the URL to each of the functions. I think you should declare as a global variable (outside of your component):
```
const apiURL = `https://api/airtable.com/v0`
```
Rather than passing it into each function as a parameter. It is a constant, that URL is never going to change for your app.